code/frontend/src/app/features/seeker-stats/upgrades-tab/upgrades-tab.component.scss

```
@@ -158,3 +142,71 @@
+
+// Table cell content
+.cell-title {
+  display: flex;
+  align-items: center;
+  gap: var(--space-2);
+  flex-wrap: wrap;
+
+  &__text {
+    font-weight: 500;
+    color: var(--text-primary);
+    word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)